deploy-manage/deploy/cloud-enterprise/install-ece-procedures.md

```diff
@@ -0,0 +1,17 @@
+---
+navigation_title: Installation procedures
+applies_to:
+  deployment:
+    ece: all
+---
+
+# ECE installation procedures
+
+Choose the Elastic Cloud Enterprise deployment scenario that best fits your business needs:
+
+* [Deploy a small installation](deploy-small-installation.md): For development, test, and small-scale use cases.
+* [Deploy a medium installation](deploy-medium-installation.md): For many production setups.
+* [Deploy a large installation](deploy-large-installation.md): For deployments with significant overall search and indexing throughput.
+* [Deploy using Podman](fresh-installation-of-ece-using-podman-hosts.md): Fresh installation of ECE using Podman hosts.
+
+For installations using Podman instead of Docker, refer to [](./fresh-installation-of-ece-using-podman-hosts.md)
```

Comment:
repeats the last bullet
```suggestion
```

Reply:
@shainaraskas : I will probably leave the last sentence and remove the Podman bullet.

The reason is that similarly to air-gapped install doc, the podman install doc is not exactly an installation guide but the considerations that need to be taken into account when installing ECE with Podman instead of Docker, so the main 3 installation guides are the "small / medium / large".

I'll try to clarify that a bit better in both air-gapped and podman install docs.